docs/md_book/src/reliability/basicis.md

```diff
@@ -1 +1,19 @@
-# Reliability
+# Introduction
+The internet is a dangerous place, and before you know it your data is gone or your data arrives duplicated because your data is split up along the way to its final destination. 
+In order to have more control over the way in which the data is transported, we have invented protocols. 
+
+In this chapter we will consider how laminar gives you more control over the transport of data.
+
+## Important
+TCP is made for reliability and does this very well. 
+We have been asked many times by people why reliability does not work well or is slow in laminar.
+Important to know is that laminar has reliability as an option but it is not focused on that and isn't made to be faster and better than TCP. 
```

Comment:
yea you are right, I tried to point out, that we don't try to have better reliability then TCP, because some people have had come up with ideas to use laminar as an FTP protocol :), Ill reference it a bit